include/SFML/Window/ContextSettings.hpp

```diff
@@ -76,6 +81,8 @@ struct ContextSettings
     unsigned int majorVersion;      ///< Major number of the context version to create
     unsigned int minorVersion;      ///< Minor number of the context version to create
     Uint32       attributeFlags;    ///< The attribute flags to create the context with
+    std::string  vendorName;        ///< Name of the vendor of the OpenGL implementation
+    std::string  rendererName;      ///< Name of the OpenGL renderer implementation
```

Comment:
What I mean is that it gives the impression that you can set them, to select the GPU (in case of dual chip) and the renderer (software/hardware, ...).

```cpp
ContextSettings settings;
settings.vendorName = "...";
settings.rendererName = "...";
window.create(..., settings);
```

This is not the same as the other members, which really select some context features -- just with the predefined default value if you don't touch them.